src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -272,6 +273,8 @@ static function ( $event ) use ( $date_object ) {
 
 			$start_of_week = get_option( 'start_of_week', 0 );
 
+			$day_url = Plugin::instance()->getLink( 'day', $day_date );
```

Comment:
This is the old way of setting up the URLs, we want to use the method:
https://github.com/moderntribe/the-events-calendar/pull/2664/files#diff-d0c94939e63f4b6608c4f83c2e99ec9cR28